modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,331 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+You can configure what gets logged, where it gets logged, and how by editing a configuration file that sits in your project.
+
+Mule logs multiple messages and specific elements in your applications' flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[`<logger>` Element] anywhere in a flow and set it up to output any message you want. By creating a configuration file, you can define what kinds of messages to log, in what way (asynchronously or synchronously), and where they get logged (such as to the console, to disk, to an endpoint or to a database).
```

Comment:
Does the first type of logging described (without the Logger) happen automatically? If so, maybe "Mule automatically logs ..."

Reply:
Tech issue: Pls double-check with the SME that you can do asynchronous logging in Mule 4. I'm talking about the "in what way (asynchronously or synchronously)" bit. Maybe it's possible within the Async scope? Not sure.